torch/csrc/jit/register_prim_ops_c10.cpp

```diff
@@ -0,0 +1,36 @@
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/ATen.h>
+#include <ATen/core/stack.h>
+
+using Stack = std::vector<c10::IValue>;
+using torch::jit::peek;
+using torch::jit::drop;
+using torch::jit::pack;
+using torch::jit::push;
+using torch::jit::pop;
+using at::Tensor;
+using at::Scalar;
+using c10::IValue;
+
+static auto registry_prim = torch::RegisterOperators().op("aten::Int.Tensor(Tensor a) -> int",
+  torch::RegisterOperators::options().kernel(c10::TensorTypeId::CPUTensorId,
```

Comment:
is there a specific reason you're doing this for `CPUTensorId` only and not as a `catchAllKernel`?

Reply:
Nope. I thought if there's tensor in argument we should use CPUTensorId. My thought can be wrong because we need to have the ops work for all kernels. I'll change it to catchAllKernel.